start.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+function startFunction {
```

Comment:
possibility to run composer install yarn and yarn build quickly would be amazing

Reply:
this should be in another PR